admin/acceptance.py

```diff
@@ -636,12 +635,15 @@ def _create_and_provision(self, reactor, name, retries=10):
         :return: Deferred that fires with the created node when
             the provisioning is completed.
         """
+        print "re-creating and re-trying node", name
+
         def create_attempt():
-            d = deferToThreadPool(
-                reactor,
-                reactor.getThreadPool(),
-                self._create_node,
-                name,
+            # A list with a single Deferred must be returned.
```

Comment:
Yes, it is kind of self-documented, but I would find it useful to have some extra info about why (ie, because it is called with a name list with only one name on it). I guess if you are familiar with the code it's perfectly self-documented, not that obvious if you have never used cloud provisioner